### PR TITLE
Change wording to be more clear about how workflow output cleanup wor…

### PR DESCRIPTION
**Pull request by dannon**
…ks.  Also adjust styling so that the rows in the workflow editor box are less smashed together.

### CODE REVIEW

File: client/galaxy/style/less/base.less [hunk 1]
```diff
@@ -671,6 +671,10 @@ div.form-row-error-message {
     margin-top: 5px;
 }
 
+.workflow-right .right-content .section-row {
```
Comment (guerler) [inline client/galaxy/style/less/base.less:674]: I actually like it how it is but do not feel too strong about it. If we really want to change this, its probably better to modify this by changing `.ui-form-element` in `ui.less`. Its currently set to `5px`. Adding `-large` to that constant will increase it to `10px` (https://github.com/galaxyproject/galaxy/blob/dev/client/galaxy/style/less/ui.less#L254).

Comment (dannon) [reply client/galaxy/style/less/base.less:674]: @guerler I did it this way so as to specifically _not_ change `.ui-form-element` across the board, only in this location.  It only looks really clumpy to me in the right pane of the workflow, hence the other class.

**Comment by MoHeydarian** [comment]
Could the 'output cleanup' option(s) include an option to permanently delete the non-starred outputs from previous steps? Currently the cleaned intermediates are deleted but still contribute to disk space. 

With a 250 Gb account limit it is challenging to analyze medium to large NGS datasets, freeing up disk space as a workflow runs would mitigate the need to monitor how much free space remains while the workflow is running. 

Since this would prevent troubleshooting a workflow, maybe a disclaimer/warning recommending to only use this option if your workflow/data is rock solid would be useful.

**Comment by dannon** [comment]
@MoHeydarian Yep, definitely.  That has been requested before, I'll add it.

**Comment by MoHeydarian** [comment]
Thanks, Dannon!

File: client/galaxy/style/less/base.less (hunk 1 above)
Comment (guerler) [inline client/galaxy/style/less/base.less]: 👍

Comment (guerler) [reply client/galaxy/style/less/base.less:674]: 👍

**Comment by dannon** [comment]
(to go ahead and get this into 16.10, I'll follow up with the purge in a separate PR)

**Comment by davebx** [comment]
@galaxybot test this

**Comment by dannon** [comment]
(resolved conflicts in 5028e2a)

**Comment by jmchilton** [comment]
Thanks @dannon!

**Pull request merged**